pages/accreditation/[slug].vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="page">
+    <section class="bg-gradient-to-b from-sushi-50 to-sushi-100 md:pt-28 max-w-screen ">
+      <div class="page-fade-in contain flex flex-col justify-center items-center max-w-8xl pt-28 pb-20 md:pb-0 lg:pt-12 gap-10">
+        <h1>{{ accred?.data.hero_title || "Your money is building a greener future" }}</h1>
+      </div>
+      <Swoosh />
+    </section>
+  </div>
+</template>
+
+<script setup lang="ts">
+/* import { asImageSrc } from '@prismicio/helpers'
+import { defineSliceZoneComponents } from '@prismicio/vue'
+import { components } from '~~/slices' */
+
+const route = useRoute()
+/* const error = ref(false) */
+
+const { client } = usePrismic()
+const slug = route.path.split('/').at(-1) ?? ''
```

Comment:
Just a silly thing here 😅 I usually access to the slug with `route.params.slug`. Like [here](https://nuxt.com/docs/getting-started/routing#route-parameters). Is there a reason why is `route.path.split('/').at(-1)` instead?